tests/check_framework/test_urls.py

```diff
@@ -35,3 +35,9 @@ def test_name_with_colon(self):
         self.assertEqual(warning.id, 'urls.W003')
         expected_msg = "Your URL pattern '^$' [name='name_with:colon'] has a name including a ':'."
         self.assertIn(expected_msg, warning.msg)
+
+    def test_no_root_urlconf_in_settings(self):
+        from django.conf import settings
+        delattr(settings, 'ROOT_URLCONF')
+        result = check_url_config(None)
```

Comment:
Make sure to override settings before altering them. See https://docs.djangoproject.com/en/1.9/topics/testing/tools/#overriding-settings